src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacReorderQuestion.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 Chris Purdy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.isaac.dos;
+
+import uk.ac.cam.cl.dtg.isaac.dos.content.DTOMapping;
+import uk.ac.cam.cl.dtg.isaac.dos.content.JsonContentType;
+import uk.ac.cam.cl.dtg.isaac.dto.IsaacReorderQuestionDTO;
+import uk.ac.cam.cl.dtg.isaac.quiz.IsaacReorderValidator;
+import uk.ac.cam.cl.dtg.isaac.quiz.ValidatesWith;
+
+
+/**
+ * Content DO for IsaacReorderQuestions.
+ *
+ */
+@DTOMapping(IsaacReorderQuestionDTO.class)
+@JsonContentType("isaacReorderQuestion")
+@ValidatesWith(IsaacReorderValidator.class)
+public class IsaacReorderQuestion extends IsaacItemQuestion { }
```

Comment:
Can this be formatted nicely, with a newline between the brackets instead of a space, and a newline at the end of the file?

Reply:
I'd argue that this is nicer formatting since it makes it very clear that the class is empty, sort of reminds me of the empty set notation `{}`. I'll change it though since it's more consistent with what's used elsewhere.

Reply:
Your IDE can do that rearranging for you, though; stylistically it's about consistency and minimising the diff if/when the class gets altered in the future.